demo/gpu-test-immediate-pod.yaml

```diff
@@ -0,0 +1,22 @@
+# Pod for gpu-test-immediate-claim.yaml
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  namespace: gpu-test-immediate
+  name: pod1
+  labels:
+    app: pod
+spec:
+  containers:
+  - name: ctr
```

Comment:
use ctr0 for consistency with other tests

Reply:
And pod0, yes, thanks.